common/config_parser/config_parser.cpp

```diff
@@ -0,0 +1,412 @@
+		if (e_cfg_int) {
+			this->update_control_map_expiration_time_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("UpdateControlMapBootExpirationTimeSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->update_control_map_boot_expiration_time_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("UpdatePollIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->update_poll_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("InventoryPollIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->inventory_poll_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("RetryPollIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->retry_poll_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("RetryPollCount");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->retry_poll_count = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptRetryTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_retry_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("StateScriptRetryIntervalSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->state_script_retry_interval_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	e_cfg_value = cfg_json.Get("ModuleTimeoutSeconds");
+	if (e_cfg_value) {
+		const json::Json value_json = e_cfg_value.value();
+		const json::ExpectedInt e_cfg_int = value_json.GetInt();
+		if (e_cfg_int) {
+			this->module_timeout_seconds = e_cfg_int.value();
+			applied = true;
+		}
+	}
+
+	/* Vectors/arrays now */
+	e_cfg_value = cfg_json.Get("ArtifactVerifyKeys");
```

Comment:
Remember to implement [this conditional](https://github.com/mendersoftware/mender/blob/28bc08ac0ed6816d96372d1c7ff50dc8d03f7a1d/conf/config.go#L265-L268).